code/modules/reagents/chemistry/reagents/blob_reagents.dm

```diff
@@ -592,75 +476,3 @@
 			for(var/mob/living/L in T)
 				L.adjust_fire_stacks(-2.5)
 				L.ExtinguishMob()
-
-//does brute damage and throws or pulls nearby objects at the target
-/datum/reagent/blob/dark_matter
-	name = "Dark Matter"
-	id = "dark_matter"
-	description = "will do medium brute damage and pull nearby objects and enemies at the target."
-	analyzerdescdamage = "Does medium brute damage and pulls nearby objects and creatures at the target."
-	color = "#61407E"
-	complementary_color = "#5D7E40"
-	message = "You feel a thrum as the blob strikes you, and everything flies at you"
-
-/datum/reagent/blob/dark_matter/reaction_mob(mob/living/M, method=TOUCH, reac_volume, show_message, touch_protection, mob/camera/blob/O)
-	reac_volume = ..()
-	reagent_vortex(M, 0, reac_volume)
-	M.apply_damage(0.3*reac_volume, BRUTE)
-
-//does brute damage and throws or pushes nearby objects away from the target
-/datum/reagent/blob/b_sorium
```

Comment:
These were the only two original blob chems I liked

Please dont remove my codebabbies